features/fixtures/maze_runner/Assets/Scripts/Scenario.cs

```diff
@@ -157,7 +157,7 @@ public bool SimpleEventCallback(IEvent @event)
         @event.AddMetadata("test1", new Dictionary<string, object> { { "test", "test" } });
         @event.AddMetadata("test2", new Dictionary<string, object> { { "test", "test" } });
         @event.ClearMetadata("test2");
-
+        @event.Unhandled = true;
```

Comment:
I'd prefer to see this tested both ways around to ensure it also works to set unhandled to handled - can we add a further scenario so we test both ways and also without any change?